src/monitor/push-metrics-for-folder.js

```diff
@@ -36,14 +36,29 @@ const pushMetricsForFolder = ({ coverageArtifactsPath, folderName, jobName, push
     const metricsPayload = getMetricsList(testStats, coverageStats)
         .reduce((metricsAsText, metric) => metricsAsText + formatMetricPayload(metric), "")
 
-    // Sent metrics to Prometheus
-    pushToGateway(`${pushGatewayUri}/metrics/job/${jobName}/folder_name/${folderName}`, metricsPayload, (err, data) => {
-        if (err) {
-            throw new Error(err.message)
-        }
+    // Sent metrics to Prometheus - try 5 times
+    const maxAttempts = 5
+    const retryDelay = 3000 // ms
```

Comment:
Maybe better to use UPPER_CASE for "real" constants for better readability.
You can also extract them in some king of configuration file (or at least extract them from the function).

Reply:
Thanks, will do that right now 😉 .

Reply:
✅  done.